atuin-client/src/result.rs

```diff
@@ -0,0 +1,11 @@
+use crate::history::History;
+
+// Return a single instance of history, but include some context.
+// EG how many other history items we have that are the same
+// Can be extended to include match context, when we use actual
+// seach engines
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct HistoryResult {
```

Comment:
I'm not super sure on the name, `Result<HistoryResult>` is... odd. But also can't think of much better, because it's like a search result but history